macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -6,7 +6,9 @@ import FlutterMacOS
 import Foundation
 
 import path_provider_foundation
+import sqflite
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
   PathProviderPlugin.register(with: registry.registrar(forPlugin: "PathProviderPlugin"))
+  SqflitePlugin.register(with: registry.registrar(forPlugin: "SqflitePlugin"))
```

Comment:
@AM5324 what is sqflite plugin do and why it is only for ios.

Reply:
It's Autogenerated I haven't used SQFLite library yet.